CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.4.4
+
+* Updates
+  * Update `vintage_net` v0.10.2
+  * Update `excoveralls` v0.14.1
+
```

Comment:
```suggestion
```

It doesn't update those. Those are in the lock file and lock files aren't distributed on hex.